core/arch/riscv/plat-spike/conf.mk

```diff
@@ -0,0 +1,76 @@
+CFG_RV64_core=y
```

Comment:
`=` is never a valid way of setting a `CFG_` variable. Your two options are `?=` and `$(call force,...)` as explained in https://optee.readthedocs.io/en/latest/building/gits/optee_os.html#configuration-variables and https://optee.readthedocs.io/en/latest/building/gits/optee_os.html#the-force-macro.